README.md

```diff
@@ -136,6 +144,9 @@ oneDNN supports systems meeting the following requirements:
 * [CMake](https://cmake.org/download/) 2.8.11 or later
 * [Doxygen](http://www.doxygen.nl/download.html#srcbin) 1.8.5 or later
   to build the documentation
+* [Arm Compute Library](https://github.com/arm-software/ComputeLibrary)
+for builds using Compute Library on AArch64, which in turn requires
+[scons](https://scons.org) to build.
```

Comment:
```suggestion
* [Arm Compute Library](https://github.com/arm-software/ComputeLibrary)
for builds using Compute Library on AArch64.
```

Probably don't need the details about scons, as those things could be changed and prebuilt binaries could be used / shipped.